recipe/meta.yaml

```diff
@@ -1,45 +1,49 @@
 {% set name = "itt-python" %}
-{% set version = "0.0.4" %}
+{% set version = "0.0.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/NERSC/itt-python/archive/36233e7dc5a4eae308defaccb4de557d93b860b9.zip
-  sha256: b3bbcb8c1b190b76310d7f3cb31c8798001b416acda9c78561c9e6a68d98a17f
-  patches:
-    - 0001-Change-vtune-to-incdir-and-ittlib-switches.patch
+  url: https://github.com/oleksandr-pavlyk/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```

Comment:
Is there a reason to use a personal repo? I would propose to create an organization or (IMHO, a better approach) to move it under some Intel-owned org.

Reply:
The `itt-python` is not an official Intel-authored package. I forked it per agreement with @rcthomas to maintain it without the need to involve him in managing the original repo NERSC/itt-python.

Intel may choose to develop its own Python API to ITT and publish it with [ittapi](https://github.com/intel/ittapi) repo the same way the Rust API is published there now.

Reply:
Okay, you're in the driver seat now then! 🚗